src/apiCalls.js

```diff
@@ -12,3 +12,26 @@ export const fetchAll = () => {
   apiDestinationsDataObj = fetchData("destinations");
   return Promise.all([apiTravelersData, apiTripsData, apiDestinationsDataObj]);
 };
+
+export const postNewTrip = (formData) => {
+  fetch("http://localhost:3001/api/v1/trips", {
```

Comment:
Great job putting this POST request together. It looks like it's well put together, and might just need to be connected to your form inputs to collect the information and POST it to the API.